scripts/create_hardlinks.py

```
@@ -1,25 +1,261 @@
+        if not dry_run:
+            logging.info("Calculating required disk space...")
+            total_size_required = 0
+            for torrent in completed_torrents:
+                content_path = Path(torrent.save_path) / torrent.name
+                if content_path.exists():
+                    if content_path.is_dir():
+                        total_size_required += _get_dir_size(content_path)
+                    else:
+                        try:
+                            total_size_required += content_path.stat().st_size
+                        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)